ci-operator/step-registry/ovn/conf/baremetal-dualstack/ovn-conf-baremetal-dualstack-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
```

Comment:
the ref is public, so I guess we can reuse `ovn-conf-baremetal-dualstack` when https://github.com/openshift/release/pull/36885 is merged.

Reply:
The plan is to re-use the ref.